dw-icon-button.js

```diff
@@ -185,18 +198,44 @@ export class DwIconButton extends buttonFocus(LitElement) {
        */
       title: { type: String },
 
+      /**
+       * Input property.
+       * Disabled Tooltip text.
+       */
+      disabledTitle: { type: String },
+
       /**
        * Input property
        * Type of the icon. By default it shows FILLED icon.
        * Possible values: FILLED and OUTLINED
        */
-      iconFont: { type: String, reflect: true }, 
+      iconFont: { type: String, reflect: true },
+
+      /**
+       * When it's `true`, shows `Material Symbols` icon.
+       * [Reference](https://fonts.google.com/icons?icon.set=Material+Symbols)
+       */
+      symbol: { type: Boolean, reflect: true },
```

Comment:
This also seems a new feature. What was the previous way to show icon?

Reply:
@ChiragMoradiya `Symbols` had been introduced by Material icons. So Whenever the user wants to use them, this property must be given. In `dw-icon` this feature was added already. So here `symbol` property is just proxied to it.